pkg/controllers/smart.go

```diff
@@ -59,6 +65,25 @@ func (controller SmartController) GetStationDetails(db *gorm.DB, w http.Response
 	json.NewEncoder(w).Encode(response)
 }
 
+func convertSDEstimate(estimate postgres.LastestEstimate) ScheduleDetail {
```

Comment:
This is currently leaving off things like the created/updated timestamps, so there are probably more fields we might want to add to the SQL query and then return here. Or, an simpler approach would be to take these IDs returned from scrapedumper and use them to look up the whole records using GORM. That would be a little less efficient though. We could also consider simplifying the signature